pages/common/ansible-galaxy.md

```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the role id for a role name `{{username.role_name}}`:
+
+`ansible-galaxy info {{username.role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```

Comment:
```suggestion
`ansible-galaxy info {{username}}.{{role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```
Actually, since the dot is mandatory it doesn't go in token syntax